index.js

```diff
@@ -282,6 +282,17 @@ function readlinkSync(path) {
   return Memory.readUtf8String(buf, n);
 }
 
+function unlinkSync(path) {
+  const { unlink } = getApi();
```

Comment:
For consistency the spaces inside the curly braces should be dropped, i.e.:

```js
  const {unlink} = getApi();
```

(I personally prefer the style you chose here, but good to remain consistent until we reformat the whole module.)